src/utils/filterTasks.js

```diff
@@ -0,0 +1,15 @@
+import { ALL, COMPLETE, INCOMPLETE } from "src/store/filter/types/filterTypes";
+
+export default function filterTasks(tasks, filterState = ALL) {
+  switch (filterState) {
+    case ALL: {
+      return tasks;
+    }
+    case INCOMPLETE: {
+      return tasks.filter((task) => task.isTaskDone === false);
+    }
+    case COMPLETE: {
+      return tasks.filter((task) => task.isTaskDone === true);
```

Comment:
Can we do like this?
```suggestion
    case INCOMPLETE: {
      return tasks.filter((task) => !task.isTaskDone);
    }
    case COMPLETE: {
      return tasks.filter((task) => task.isTaskDone);
```